package.json

```diff
@@ -8,14 +8,17 @@
     "build": "gulp",
     "build:dist": "gulp --dist",
     "start": "nodemon dev/server/server.js",
-    "server:dist": "node dist/server/server.js"
+    "server:dist": "node dist/server/server.js",
+    "test": "./node_modules/.bin/mocha --compilers js:babel-register --bail --reporter nyan ./src/specs/**/*.js"
```

Comment:
You just need `mocha` here. If the package is installed locally, it'll use `./node_modules/.bin/mocha` automatically.